### PR TITLE
[FIX] Refer to #15165 issue for the description of the problem this PR solves

### PR DESCRIPTION
**Pull request by daramousk**
#15165
--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/payment/models/payment_acquirer.py [hunk 1]
```diff
@@ -429,7 +429,7 @@ def _get_default_partner_country_id(self):
     # duplicate partner / transaction data to store the values at transaction time
     partner_id = fields.Many2one('res.partner', 'Partner', track_visibility='onchange')
     partner_name = fields.Char('Partner Name')
-    partner_lang = fields.Selection(_lang_get, 'Language', default='en_US')
```
Comment (mart-e) [inline addons/payment/models/payment_acquirer.py]: on [partner](https://github.com/odoo/odoo/blob/79bd2f3062b039d39b6f0f0259a9e8efae08acc8/odoo/addons/base/res/res_partner.py#L149), we use `default=lambda self: self.env.lang` Have you tried it?

File: doc/cla/individual/daramousk.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+
+I hereby agree to the terms of the Odoo Individual Contributor License
+Agreement v1.0.
+
+I declare that I am authorized and able to make this agreement and sign this
+declaration.
+
+Signed,
+
+George Daramouskas george.daramouskas@gmail.com https://github.com/daramousk
```
Comment (mart-e) [inline doc/cla/individual/daramousk.md]: Thank you for the CLA signature but it seems there is still an issue.

- you are signing using the address `g...@targetintegration.com`
- you are signing for the address `g...@gmail.com`

The email address in the CLA must be the same as for the signature for the verification to succeed.
You can either change the your [git configuration](https://help.github.com/articles/setting-your-email-in-git/) or modify the email in the CLA signature.

**Comment by daramousk** [comment]
Updated the pull request @mart-e .

Thank you for your feedback.

**Comment by mart-e** [comment]
Thanks for the quick update, merged in 10.0

**Pull request merged**